src/patternfly/components/FormControl/form-control.scss

```
@@ -268,17 +269,17 @@
   &.pf-m-placeholder > select {
     --#{$form-control}--Color: var(--#{$form-control}--m-placeholder--Color);
 
-    > * {
+    * {
       --#{$form-control}--Color: var(--#{$form-control}--m-placeholder--child--Color);
 
       // stylelint-disable max-nesting-depth
       &:disabled {
         color: revert;
       }
       // stylelint-enable
-    }
-  }
-  
+    }      
+  }  
```

nit - looks like some extra whitespace was added here

```suggestion
    }
  }
```